app/views/organizations/staff/people/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="col-md-8 mx-auto card p-5 bigger">
+    <%= bootstrap_form_with model: [:staff, person] do |form| %>
+
+      <%= form.hidden_field :user_id,
+                            value: current_user.id %>
+
+       <div class='form-group'>
+        <%= form.text_field :email, value: Current.user.email, readonly: true, class: 'form-control' %>
```

Comment:
for all cases we use `Current.user` in a controller or view, we can change to `current_user` as it's more conventional in Rails. Devise is so ubiquitous it's a common method to refer to.